### DISCUSSION TITLE
New tabs to be opened on the left side while older tabs generally be aligned to the right?

### DISCUSSION THREAD

**Discussion by AjvarXX** [onemen/TabMixPlus#108] [Ideas]
Greetings. I couldn't register on the tabmixplus forum due to Confirmation code pic doesn't load so here I am here.

Could you make tabs to order new to older from left to right instead of right to left?

**Comment by onemen** [comment]
if anyone would like to work on this, open a pull request with the code

**Comment by 117649** [comment]
> Greetings. I couldn't register on the tabmixplus forum due to Confirmation code pic doesn't load so here I am here.
> 
> Could you make tabs to order new to older from left to right instead of right to left?

Are you mean this? 
![image](https://user-images.githubusercontent.com/8281345/151797731-f0b012ee-0458-4ba6-bae9-78c6a66fc67d.png)

**Reply by AjvarXX** [reply]
Nope. Meaning like 3 2 1 c b a.
